zerver/openapi/zulip.yaml

```diff
@@ -9459,7 +9459,7 @@ paths:
             When enabled, clients should display the names of reacting users, rather than
             a count, for messages with few total reactions. The ideal cutoff may depend on
             the space available for displaying reactions; the official web application
-            displays names when <=3 total reactions are present with this setting enabled.
+            displays names when 3 or less total reactions are present with this setting enabled.
```

Comment:
We could also say "between 1 and 3 total reactions" for these, but that felt wordier.

Reply:
Should be “fewer” rather than “less”.

Reply:
Fixed.